includes/abstracts/abstract.llms.payment.gateway.php

```diff
@@ -135,19 +135,33 @@ public function complete_transaction( $order, $deprecated = '' ) {
 
 		$this->log( $this->get_admin_title() . ' `complete_transaction()` started', $order );
 
-		// redirect to the product's permalink
-		$redirect = get_permalink( $order->get( 'product_id' ) );
+		// get the redirect parameter.
+		$redirect = urldecode( filter_input( INPUT_GET, 'redirect', FILTER_VALIDATE_URL ) );
```

Comment:
I copied this from https://github.com/gocodebox/lifterlms/blob/5f48ab6d09863d8688d07dd24753c4bd8d830367/includes/controllers/class.llms.controller.orders.php#L224-L229. Fixed this and created a new issue to fix all instances of superglobals on `class.llms.controller.orders.php` so that I can fix them as well in the next iteration: https://github.com/gocodebox/lifterlms/issues/788